tldr/dfg_extractor.py

```
@@ -3738,3 +3745,169 @@ def extract_luau_dfg(code: str, function_name: str) -> DFGInfo:
         var_refs=visitor.refs,
         dataflow_edges=edges,
     )
+
+
+# =============================================================================
+# Zig DFG Extraction
+# =============================================================================
+
+class ZigDefUseVisitor:
+
+    def __init__(self, source: bytes):
```

Ruff (0.14.14)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)